Controller/AuthorizeController.php

```diff
@@ -152,7 +152,8 @@ protected function getClient()
             if (null !== $request) {
                 if (null === $clientId = $request->get('client_id')) {
                     $form = $this->container->get('fos_oauth_server.authorize.form');
-                    $clientId = $request->get(sprintf('%s[client_id]', $form->getName()), null, true);
+                    $formData = $request->get($form->getName(), []);
```

Comment:
`[]` is not compatible with PHP version `<5.4`. This bundle require PHP `>=5.3.3`. You must use `array()`.